hera_cal/lst_stack/calibration.py

```diff
@@ -132,240 +174,350 @@ def lstbin_absolute_calibration(
+            number of baselines, frequencies, and polarizations as the data in stack. The model
+            can simply be the mean of the data in the stack if data are already abscal'd.
+        all_reds : list of list of tuples
+            A list of lists of redundant baseline groups. Each element of the list is a list of tuples
+            containing the redundant baseline groups. Each tuple contains the antenna numbers
+            of the antennas in the redundant baseline group.
+        inpaint_bands : tuple of slices, default=(slice(0, None, None),)
+            Defines the frequency bands to use for smoothing. Each slice object in the tuple
+            defines a separate frequency band to use for smoothing. The default is to smooth over
+            all frequencies at once.
+        auto_stack : LSTStack, default=None
+            An LSTStack object containing the stack of auto-correlations. If provided, the
+            auto-correlations will be calibrated using the same gains calculated from the
+            cross-correlations. Note that the auto-correlations are used to calculate the gain
+            amplitude if use_autos_for_abscal is set to True.
+        auto_model : np.ndarray, default=None
+            The reference model to calibrate the auto-correlations to. The model should have the
+            same number of baselines, frequencies, and polarizations as the auto-correlations in
+            auto_stack.
+        run_amplitude_cal : bool, default=True
+            Boolean flag to run amplitude calibration.
+        run_phase_cal : bool, default=True
+            Boolean flag to run tip-tilt calibration.
+        smoothing_scale : float, default=10e6
+            The scale of the smoothing function used to smooth the gains. This is the width of the
+            smoothing function in Hz.
+        smooth_gains : bool, default=True
+            Boolean flag to smooth the gains.
+        eigenval_cutoff : float, default=1e-12
+            The cutoff for the eigenvalues of the DPSS eigenvectors.
+        calibrate_inplace : bool, default=True
+            Boolean flag to calibrate the data in place.
+        use_autos_for_abscal : bool, default=True
+            Boolean flag to use the auto-correlations for absolute calibration. If set to True,
+            the auto-correlations will be used to calculate the gain amplitude if they are provided.
+            Note that the auto_model must also be provided if this flag is set to True and autos are
+            provided. If set to False, the auto-correlations will not be used for calibration.
+
+    Returns:
+    -------
+        calibration_parameters : dict
+            A dictionary containing the calibration parameters. The keys are as follows:
+                - 'A_J{pol}' : The amplitude calibration parameters for each polarization.
+                - 'T_J{pol}' : The tip-tilt calibration parameters for each polarization
```

Comment:
I'd clarify the shape and type of these dictionary entires

Reply:
Clarified this